cluster/manifests/ingress-controller/deployment.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: kube-system
   labels:
     application: kube-ingress-aws-controller
-    version: v0.11.7
```

Comment:
what are other changes between 0.11.7 and 0.11.10?

Reply:
[v0.11.9](https://github.com/zalando-incubator/kube-ingress-aws-controller/releases/tag/v0.11.9) is a noop (fixes Github actions)
[v0.11.8](https://github.com/zalando-incubator/kube-ingress-aws-controller/releases/tag/v0.11.8) are dependency updates reported by dependabot.